docs/next.config.js

```diff
@@ -13,6 +13,10 @@ const workspaceRoot = path.join(__dirname, '../');
 const reactMode = 'legacy';
 
 module.exports = {
+  typescript: {
+    // Motivated by https://github.com/zeit/next.js/issues/7687
+    ignoreDevErrors: true,
```

Comment:
We should add `ignoreBuildErrors` as well. Typechecking is also done separately as well. This should increase time-to-deploy (and bundle size comments).

Reply:
Great point